### PR TITLE
Add Peru holidays

### PR DESCRIPTION
**Pull request by dlopez525**

### CODE REVIEW

File: src/Countries/Peru.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+use Carbon\CarbonImmutable;
+
+class Peru extends Country
+{
+    public function countryCode(): string
+    {
+        return 'pe';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Peru.php:14]: can be removed

File: src/Countries/Peru.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            'Inmaculada Concepción' => '12-08',
+            'Batalla de Ayacucho' => '12-09',
+            'Navidad' => '12-25',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
```
Comment (Nielsvanpach) [inline src/Countries/Peru.php:38]: use `$this->easter($year)` instead. Available after rebasing your branch from this repo's main branch.

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you rebase, so we can run the tests here?

**Comment by dlopez525** [comment]
> Thanks! Could you rebase, so we can run the tests here?

I'm sorry, I can't understand this. :/

**Comment by dlopez525** [comment]
> Thanks! Could you rebase, so we can run the tests here?

uhmm ready...I'll stay tuned

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**